docs/components/roadmap/dependencies-graph.md

```
@@ -0,0 +1,83 @@
+<!-- Generated by docs-components; do not edit manually. -->
+```mermaid
```

markdownlint-cli2 (0.18.1)

First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)